app/models/mixins/retirement_mixin.rb

```diff
@@ -253,15 +253,18 @@ def valid_zone?
   end
 
   def system_context_requester
-    if try(:evm_owner_id).present?
-      if User.find_by(:id => evm_owner_id).present?
-        return evm_owner
-      else
-        $log.info("#{name} has evm_owner_id present but no user with that id found so defaulting to admin.")
-        return User.super_admin
+    user = User.find_by(:id => evm_owner_id) if try(:evm_owner_id)
```

Comment:
Why is `if try(:evm_owner_id)` necessary?

Reply:
Because the data migration didn't isn't present on this version so we have to have this check.

Reply:
Also I don't understand why we're being more nitpicky on this PR than we were on the version that got into master.

Reply:
`respond_to?` should be used in that case.